### ISSUE TITLE
Allow setting a different price depending on weather conditions

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#3007]
On some cities, weather conditions should trigger a change on the price of the delivery. 

For example, in Zaragoza, when it's windy, it can be dangerous to drive the bike. 

This can be implemented with a "global" setting on orders dashboard, that can be enabled/disabled by the dispatcher. 
Two buttons, "Rain" and "Wind" should be ok. 

Also, there should be a condition in pricing rules for weather conditions.

**Comment by ERAMANrepartos** [comment]
I don't think this should be assumed to be a good feature to implement. Rather, this makes customers feel guilty for their purchase. I'm also not sure that "pay me 10% more" is a good political response to the dangerous conditions we find ourselves in.
